authentication/src/main/java/io/camunda/authentication/service/BasicCamundaUserService.java

```diff
@@ -82,4 +89,19 @@ protected List<String> getAuthorizedApplications() {
         ? applicationAccess.authorization().resourceIds()
         : List.of();
   }
+
+  private List<TenantEntity> getTenantsForUser(final CamundaUser camundaUser) {
```

Comment:
FYI: This will be consolidated with https://github.com/camunda/camunda/issues/36194 to avoid duplication.